// ng2-ckeditor/esm2020/ckeditor.module.mjs
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { CKEditorComponent } from './ckeditor.component';
import { CKButtonDirective } from './ckbutton.directive';
import { CKGroupDirective } from './ckgroup.directive';
import * as i0 from "@angular/core";
/**
 * CKEditorModule
 */
export class CKEditorModule {
}
CKEditorModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.0.2", ngImport: i0, type: CKEditorModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
CKEditorModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.0.2", ngImport: i0, type: CKEditorModule, declarations: [CKEditorComponent, CKButtonDirective, CKGroupDirective], imports: [CommonModule], exports: [CKEditorComponent, CKButtonDirective, CKGroupDirective] });
CKEditorModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.0.2", ngImport: i0, type: CKEditorModule, imports: [[CommonModule]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.0.2", ngImport: i0, type: CKEditorModule, decorators: [{
            type: NgModule,
            args: [{
                    imports: [CommonModule],
                    declarations: [CKEditorComponent, CKButtonDirective, CKGroupDirective],
                    exports: [CKEditorComponent, CKButtonDirective, CKGroupDirective],
                }]
        }] });
